gtsfm/loader/astronet_loader.py

```diff
@@ -118,13 +122,19 @@ def __init__(
         self._num_imgs = len(self._image_paths)
         logger.info("AstroNet loader found and loaded %d images and %d tracks.", self._num_imgs, self.num_sfmtracks)
 
+    @property
+    def gt_gtsfm_data(self):
+        return self._gt_gtsfm_data
+
     @staticmethod
     def colmap2gtsfm(
```

Comment:
why is this a static method in this class? like what does it have to do with astronet specifically?

Reply:
AstroNet data is in COLMAP format. I could move this to the base class since it would also be useful for the Colmap data?

Reply:
I dont feel like it is related to the loader, maybe it can be moved to a util file. Why would a user of a loader want to convert colmap data to gtsfm format? I would assume it just provides the gtsfm format, using whatever underneath.

Reply:
Because the AstroNet data is in Colmap format and it needs to be converted to GTSfM format before it can be used by the pipeline.

Reply:
I mean the loader is supposed to abstract that away. I think this PR https://github.com/borglab/gtsfm/pull/346 is moving it to a util? Which is better I think

Reply:
The loader can call a util, convert it to GTSfM format and provide it, the user of the loader does not have to know what format Astronet data is natively in.

Reply:
**The loader is responsible for reading in the specific data and converting it to a format usable by GTSfM.** 

I don't think it makes sense to have to add a util in some other file for every dataset used, as this will just congest the other modules. However, I think it's fine to move this specific function to a util, as we work with the Colmap format a lot.

When other users want to apply GTSfM to their specific data, all they should need to do is fill in a new loader (and maybe a runner) without touching any other code. It's infeasible to expect our main code base to handle converting _every single_ dataset to the expected GTSfM format.

Reply:
> However, I think it's fine to move this specific function to a util, as we work with the Colmap format a lot.

 yes, that is my reason as well. 
I don't see why anyone would use the Astronet loader to convert colmap data to gtsfm. It already provides data in gtsfm format. And even if we did want to convert colmap to gtsfm format for some other reason, it does not need a AstroNet loader.

Reply:
https://github.com/borglab/gtsfm/pull/346 is trying to move it to io_utils.py, which I think is better than a static method in AstroNet loader (my point that its being used in a non-loader specific context).

Reply:
Maybe I should just leave it as is here to avoid conflicts with #346?

Reply:
SInce youre changing the API and Jon isn't I think we should merge this is first. 

@womackj1 Can #346 wait until this is merged? I don't see any updates on it recently.